mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/1d-depthwise-conv.mlir

```diff
@@ -0,0 +1,60 @@
+// DEFINE: %{compile} =  mlir-opt %s \
+// DEFINE:    -transform-interpreter -test-transform-dialect-erase-schedule \
+// DEFINE:    -one-shot-bufferize="bufferize-function-boundaries" -lower-vector-mask -cse -canonicalize -convert-vector-to-scf -arm-sve-legalize-vector-storage \
+// DEFINE:    -convert-vector-to-llvm="enable-arm-sve" -test-lower-to-llvm -o %t
+// DEFINE: %{entry_point} = conv
+// DEFINE: %{run} = %mcr_aarch64_cmd %t -e %{entry_point} -entry-point-result=void --march=aarch64 --mattr="+sve"\
+// DEFINE:    -shared-libs=%mlir_runner_utils,%mlir_c_runner_utils
+
+// RUN: %{compile} | %{run} | FileCheck %s
```

Comment:
Here you've both using input + output files (via `-o`), but also piping things. This seems to work when `%mcr_aarch64_cmd` is an emulator, but if you replace it with `mlir-cpu-runner` (on SVE hardware) the test fails on the first run (but works on the second run).

Reply:
P.s. `-convert-vector-to-scf -arm-sve-legalize-vector-storage  -convert-vector-to-llvm="enable-arm-sve"` can be replaced with `-convert-vector-to-scf=full-unroll`.

Reply:
Thanks, good catch! 

Yeah, the `RUN` should be something like this: `// RUN: rm -f %t && %{compile} && %{run} | FileCheck %s` instead. However, the current `RUN` line, when fails, produces:
```
# .---command stderr------------
# | cannot open input file '/llvm-project/build/release_vls/tools/mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/Output/1d-depthwise-conv.mlir.tmp': No such file or directory
# | could not parse the input IR
# `-----------------------------
```

rather than (from https://lab.llvm.org/buildbot/#/builders/176/builds/9331):
```
# .---command stderr------------
# | Error: entry point not found
# `-----------------------------
```

Reply:
A guess: It may depend on the shell/system if the result is an empty file or no file (`Error: entry point not found` seems to indicate the input was empty).

Reply:
That could be the case 🤔

There's https://github.com/llvm/llvm-project/blob/main/mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/matmul.mlir that works fine with this configuration - we probably just missed that it failed first time it was run. Also a guess :)

If I don't hear back from Omair with other suggestion, I'll merge the updated version around Monday and just monitor the bots to see what happens. Thanks Ben!

Reply:
No worries, sounds good to me :) 

In the matmul test there's a lone `// RUN: %{compile}`, with the run steps separate, which would not have this issue.